bark/world/objects/agent.cpp

```diff
@@ -16,6 +16,11 @@ namespace bark {
 namespace world {
 namespace objects {
 
+enum AgentStatus : unsigned int {
```

Comment:
as mentioned before, should we not handle that in the behavior model.
The behavior model sets INVALID, the agent does not move.
Seems redundant

Reply:
your're right, I removed it

Reply:
I removed the enum